azure-pipelines.yml

```diff
@@ -227,29 +227,6 @@ jobs:
     condition: succeededOrFailed()
     continueOnError: True
 
-- job: chip_englishbreakfast_verilator
-  displayName: Verilated English Breakfast (Build)
-  # Build Verilator simulation of the English Breakfast toplevel design
-  dependsOn: lint
-  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'), eq(dependencies.lint.outputs['DetermineBuildType.onlyCdcChanges'], '0'))
```

Comment:
Do we want to have something similar in GHA?

Reply:
Maybe, but this basically never worked properly in Azure and didn't save us any time. I think our time would be better spent getting Bazel caching working properly